docs/sriov-secondary-interface.md

```diff
@@ -0,0 +1,54 @@
+# SR-IOV Secondary Interface Support
```

Comment:
I feel we should merge the doc after the implementation is complete - when users can really create a secondary SR-IOV interface. If we merge the doc with this PR, readers will think they can follow the doc to create a secondary interface already, but it wont work.

Reply:
Sure; the doc can merge later. Thanks for the suggestions.